source/features/expand-urls-remove-utms.js

```diff
@@ -0,0 +1,13 @@
+function removeUTMs(url) {
+	return url.replace(/[?&#]utm_.*/, '');
```

Comment:
It would be better to extract use the `URL` and `URLSearchParams` APIs to handle the query strings properly. Naive replacements like this is fragile.